lib/data/external/AwsClient.js

```diff
@@ -208,6 +208,22 @@ class AwsClient {
             });
         });
     }
+
+    abortMPU(key, uploadId, location, bucket, log, callback) {
+        const awsBucket = this._awsBucketName;
+        const awsKey = this._createAwsKey(bucket, key, this._bucketMatch);
+        const abortParams = {
+            Bucket: awsBucket, Key: awsKey, UploadId: uploadId,
+        };
+        return this._client.abortMultipartUpload(abortParams, err => {
+            if (err) {
+                log.trace('err from data backend on abortMPU',
+                { error: err, dataStoreNae: this._dataStoreName });
```

Comment:
can you please make sure the actual error is showing up in the logs?  we might have to do an err.message for the error to actually appear.